package.json

```diff
@@ -14,18 +14,17 @@
         "format": "d2-style apply",
         "validate-commit": "d2-style check --staged",
         "validate-push": "yarn test",
-        "cy:open": "start-server-and-test 'yarn start' http://localhost:3000 'yarn cypress open --env networkMode=live'",
+        "cy:open": "start-server-and-test 'yarn start' http://localhost:3000 'yarn cypress open --e2e --browser electron --env networkMode=live'",
```

Comment:
Since Cypress now offers both e2e and component testing, the UI starts with a prompt which lets you decide which of these test types you want. By adding `--e2e` we skip this prompt.

By adding `--browser electron`, the tests start automatically in the integrated electron window. Running the tests in Chrome causes CORS issues, so causes all tests to fail. Seems we're stuck on Electron when using the Cypress UI, which shouldn't be a problem since it's also Chromium, and the more important Cypress runs _do_ use headless Chrome.